FitVid/src/index.js

```diff
@@ -11,29 +11,32 @@ import { HistoryProvider } from "./Context/history-context";
 import { WatchLaterProvider } from "./Context/watch-later-context";
 import { PlayListModalProvider } from "./Context/playlist-modal-context";
 import { PlayListsProvider } from "./Context/playlist-context";
+import { CategoryProvider } from "./Context/category-context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <PlayListsProvider>
-      <PlayListModalProvider>
-        <WatchLaterProvider>
-          <HistoryProvider>
-            <LikedProvider>
-              <UserProvider>
-                <ToastProvider>
-                  <Router>
-                    <App />
-                  </Router>
-                </ToastProvider>
-              </UserProvider>
-            </LikedProvider>
-          </HistoryProvider>
-        </WatchLaterProvider>
-      </PlayListModalProvider>
-    </PlayListsProvider>
+    <CategoryProvider>
+      <PlayListsProvider>
+        <PlayListModalProvider>
+          <WatchLaterProvider>
+            <HistoryProvider>
+              <LikedProvider>
+                <UserProvider>
+                  <ToastProvider>
+                    <Router>
+                      <App />
+                    </Router>
+                  </ToastProvider>
+                </UserProvider>
```

Comment:
Suggestion : you can place Router at the top as pe rthe best practice

Reply:
Done